stdlib/3/optparse.pyi

```diff
@@ -0,0 +1,248 @@
+    values = ...  # type: Any
+    version = ...  # type: str
+    def __init__(self, *args, **kwargs) -> None: ...
+    def _add_help_option(self) -> None: ...
+    def _add_version_option(self) -> None: ...
+    def _create_option_list(self) -> None: ...
+    def _get_all_options(self) -> List[Any]: ...
+    def _get_args(self, args: Iterable) -> List[Any]: ...
+    def _get_encoding(self, file) -> Any: ...
+    def _init_parsing_state(self) -> None: ...
+    def _match_long_opt(self, opt) -> Any: ...
+    def _populate_option_list(self, option_list, *args, **kwargs) -> None: ...
+    def _process_args(self, largs: List[str], rargs: List, values: Values) -> None: ...
+    def _process_long_opt(self, rargs: List, values) -> None: ...
+    def _process_short_opts(self, rargs: List, values) -> None: ...
+    def add_option_group(self, *args, **kwargs) -> OptionParser: ...
+    def check_values(self, values, args) -> Tuple[Any, ...]: ...
+    def disable_interspersed_args(self) -> None: ...
+    def enable_interspersed_args(self) -> None: ...
+    def error(self, msg) -> None: ...
+    def exit(self, *args, **kwargs) -> None: ...
+    def expand_prog_name(self, s: Optional[str]) -> Any: ...
+    def format_epilog(self, formatter: Union[HelpFormatter, OptionParser, None]) -> Any: ...
+    def format_help(self, *args, **kwargs) -> str: ...
+    def format_option_help(self, *args, **kwargs) -> str: ...
+    def get_default_values(self) -> Values: ...
+    def get_option_group(self, opt_str) -> Any: ...
+    def get_prog_name(self) -> Any: ...
+    def get_usage(self) -> str: ...
+    def get_version(self) -> Any: ...
+    def parse_args(self, *args, **kwargs) -> Tuple[Any, ...]: ...
+    def print_help(self, *args, **kwargs) -> None: ...
+    def print_usage(self, *args, **kwargs) -> None: ...
+    def print_version(self, *args, **kwargs) -> None: ...
+    def set_default(self, dest, value) -> None: ...
+    def set_defaults(self, *args, **kwargs) -> None: ...
+    def set_process_default_values(self, process) -> None: ...
+    def set_usage(self, usage: str) -> None: ...
+
+class OptionValueError(OptParseError):
+    msg = ...  # type: Any
+
+class TitledHelpFormatter(HelpFormatter):
+    _long_opt_fmt = ...  # type: str
+    _short_opt_fmt = ...  # type: str
+    current_indent = ...  # type: int
+    default_tag = ...  # type: str
+    help_position = ...  # type: int
+    help_width = ...  # type: None
+    indent_increment = ...  # type: Any
+    level = ...  # type: int
+    max_help_position = ...  # type: int
+    option_strings = ...  # type: Dict
+    parser = ...  # type: None
+    short_first = ...  # type: Any
+    width = ...  # type: Any
+    def __init__(self, *args, **kwargs) -> None: ...
+    def format_heading(self, heading) -> str: ...
+    def format_usage(self, usage) -> str: ...
+
+class Values:
+    def __cmp__(self, other) -> int: ...
+    def __init__(self, *args, **kwargs) -> None: ...
```

Comment:
I think this should be `def __init__(self, defaults: Optional[Mapping[str, Any]] = ...) -> None: ...`